### PR TITLE
Add floating ips and floating ip actions support.

### PR DESCRIPTION
**Pull request by phillbaker**
Adds support for upcoming floating ip endpoints in API v2.

- [x] Depends on  a75f8e7 in https://github.com/digitalocean/droplet_kit/pull/67 to fix unrelated spec failures.

cc @nanzhong

### CODE REVIEW

**Comment by nanzhong** [comment]
:+1:

**Comment by andrewsomething** [comment]
@phillbaker Shouldn't it also get added to `self.resources` in `lib/droplet_kit/client.rb` ?

File: spec/lib/droplet_kit/resources/floating_ip_action_resource_spec.rb [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    end
+  end
+
+  describe '#unassign' do
+    let(:ip) { '127.0.0.1' }
+    let(:path) { "/v2/floating_ips/#{ip}/actions" }
+    let(:action) { 'unassign' }
+    let(:fixture) { api_fixture("floating_ip_actions/#{action}") }
+
+    it 'sends an assign request for a floating ip' do
```
Comment (jademcgough) [inline spec/lib/droplet_kit/resources/floating_ip_action_resource_spec.rb]: this should be `sends an unassign request for a floating ip`

Comment (phillbaker) [inline spec/lib/droplet_kit/resources/floating_ip_action_resource_spec.rb]: Thanks!

**Comment by nanzhong** [comment]
:ship:

**Pull request merged**